1.5.0

```diff
@@ -687,6 +687,344 @@
           }
         }
       ]
+    },
+    {
+      "sObjectName": "PersonAccount",
+      "instructionTypeName": "command__Client",
+      "taskList": [
+        {
+          "jiraTicket": "ac-9548a",
+          "description": "Add Alert Message Field to Person Account Client Layout",
+          "actionType": "Field",
+          "actionTypeName": "command__Alert_Message__c",
+          "location": {
+            "position": "Below",
+            "reference": "Description",
+            "section": "Description Information",
+            "column": 0,
+            "behavior": "Edit"
+          }
+        }
+      ]
+    },
+    {
+      "sObjectName": "Lead",
+      "instructionTypeName": "command__AC Lead Layout",
+      "taskList": [
+        {
+          "jiraTicket": "ac-9548b",
+          "description": "Add Alert Message Field to Lead Layout",
+          "actionType": "Field",
+          "actionTypeName": "command__Alert_Message__c",
+          "location": {
+            "position": "Below",
+            "reference": "Description",
+            "section": "Description Information",
+            "column": 0,
+            "behavior": "Edit"
+          }
+        }
+      ]
+    },
+    {
+      "instructionTypeName": "FlexiPageComponent",
+      "taskList": [
+        {
+          "jiraTicket": "AC-9548c1",
+          "actionType": "FlexiPageComponent",
+          "actionTypeName": "FlexiPageComponent",
```

Comment:
Isn't this going to add it to cloned page layouts? I thought we aren't doing that for anything other than the new notes component.